integration_tests/tests/features/claim_fees.feature

```diff
@@ -84,52 +84,3 @@ Feature: Claim Fees
 
     # Claim fees into ACC2
     When I claim fees for validator VN and epoch 1 into account ACC2 using the wallet daemon WALLET_D, it fails
-
-  @serial
-  Scenario: Prevent validator fees claim for unauthorized wallet
```

Comment:
I deleted this test as it was basically the same as the one above

Reply:
This test above checks that the fees can't be double-claimed. This test checks if a wallet does not use/possess the private claim key, it cannot claim fees.

Reply:
Ah ok, thanks. I've added a line to the test to also check that a non-account holder can't claim the fees